unity-renderer/Assets/Scripts/MainScripts/DCL/Components/Textures/DCLTexture.cs

```diff
@@ -206,7 +206,7 @@ public override void Dispose()
                 RemoveReference(attachedEntitiesByComponent.First().Key);
             }
 
-            if (texturePromise != null)
+            if (texturePromise != null && AssetPromiseKeeper_Texture.i != null)
```

Comment:
`AssetPromiseKeeper_Texture.i` should never be null, it's auto instantiated when requested.

Reply:
Thanks, it seemed to make a test fail at some point, but it wasn't.